### PR TITLE
Feature/add module rating - part 1 of 2

### PR DESCRIPTION
**Pull request by GeeH**

### CODE REVIEW

File: bin/cli.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+chdir(__DIR__ . '/../');
+require 'vendor/autoload.php'; // grabs the Composer autoloader
+error_reporting(E_ALL ^ E_WARNING);
+set_time_limit(0);
+
+$application = Zend\Mvc\Application::init(require 'config/application.config.php');
+
+$services = $application->getServiceManager();
+
+$updateModel = $services->get(Update::class);
```
Comment (localheinz) [inline bin/cli.php]: @GeeH 

Unsure about the naming here - I understand `ZfModule\Service\Update` as a service, not a model?

Comment (GeeH) [inline bin/cli.php]: It's not a model, it's a service.

File: module/ZfModule/src/ZfModule/Service/Update.php [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        $module->setUpdatedAt(date('Y-m-d H:i:s'));
+
+        $this->moduleMapper->update($module);
+    }
+
+    /**
+     * @param ModuleEntity $module
+     * @return array|bool
+     */
+    public function getFromGithub(ModuleEntity $module)
```
Comment (ins0) [inline module/ZfModule/src/ZfModule/Service/Update.php]: wouldn't it be better to handle this stuff in the ``RepositoryRetreiver`` ?

File: module/ZfModule/src/ZfModule/Service/Update.php [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+        }
+
+        if (!is_array($data)) {
+            return false;
+        }
+
+        return $data;
+    }
+
+    private function setupClient()
```
Comment (ins0) [inline module/ZfModule/src/ZfModule/Service/Update.php]: see above

Comment (GeeH) [inline module/ZfModule/src/ZfModule/Service/Update.php]: Nah - EdpGithub has a dependency on ScnSocialAuth which has a dependency on some omni-auth component. Long and the short is it only works with Http requests and I cannot be bothered to fix that.

**Comment by localheinz** [comment]
@GeeH 

Not sure, what is the state of this, shall we close it, or would you prefer to continue working on this?

**Comment by GeeH** [comment]
@localheinz 

No idea, if I'm honest, I think this whole project is a shitshow that probably needs to be either rewritten from scratch, or shut down.

**Comment by Ocramius** [comment]
Agree here: serves no purpose on its own, even more now that packagist supports most of what we'd look for in it.

**Comment by localheinz** [comment]
@GeeH @Ocramius 

Not sure, should we shut the site down, then?

**Comment by Ocramius** [comment]
I think so. Maybe freeze it into static mode, to preserve links.

**Pull request closed without merging**